test/e2e/adminUI/pages/fieldTypes/datetime.js

```diff
@@ -0,0 +1,56 @@
+var utils = require('../../../utils');
+
+module.exports = function DatetimeType(config) {
+	var self = {
+		selector: '.field-type-datetime', //There should be a [for="' + config.fieldName + '"] here, but this isn't in the html.
+		elements: {
+			label: '.FormLabel',
+			nowButton: '.Button--default',
+			date: 'input[name="' + config.fieldName + '_date"]',
+			datePlaceholder: 'input[placeholder="YYYY-MM-DD"]',
+			time: 'input[name="' + config.fieldName + '_time"]',
+			timePlaceholder: 'input[placeholder="HH:MM:SS am/pm"]',
+		},
+		commands: [{
+			verifyUI: function() {
+				this
+					.expect.element('@label').to.be.visible;
+				this
+					.expect.element('@label').text.to.equal(utils.titlecase(config.fieldName));
+				this
+					.expect.element('@nowButton').to.be.visible;
+				this
+					.expect.element('@date').to.be.visible;
+				this
+					.expect.element('@datePlaceholder').to.be.visible;
+				this
+					.expect.element('@time').to.be.visible;
+				this
+					.expect.element('@timePlaceholder').to.be.visible;
+				return this;
+			},
+			fillInput: function(input) {
+				this
+					.clearValue('@date')
+					.setValue('@date', input.date)
+					.clearValue('@time')
+					.setValue('@time', input.time)
+
+				return this;
+			},
+			verifyInput: function(input) {
+				this
+					.getValue('@date', function(result) {
+						this.api.assert.equal(result.value, input.date);
+					});
+				this
+					.getValue('@time', function(result) {
+						this.api.assert.equal(result.value, input.time);
+					});
+				return this;
```

Comment:
The tests seem inconsistent about whether line 50s `return this;` is included in the `verifyInput` function. 

I think perhaps they should, and it seems to be the ones I have done that are occasionally missing it. Surprised this didn't cause problems?